app/controllers/report_controller/reports/editor.rb

```diff
@@ -651,12 +651,6 @@ def gfv_chargeback
       @edit[:new][:cb_tenant_id] = params[:cb_tenant_id].blank? ? nil : params[:cb_tenant_id].to_i
     elsif params.key?(:cb_tag_value)
       @edit[:new][:cb_tag_value] = params[:cb_tag_value].blank? ? nil : params[:cb_tag_value]
-    elsif params.key?(:cb_model)
```

Comment:
@zeari I did not find why there is this if-leg - is there a case when it is triggered ?
addded here https://github.com/ManageIQ/manageiq/commit/82f4b9fb36d382d8f1b43be10a5e4dc2ea50afd5#diff-a1e3f16734c9d90e5b6e866305eab359R635

thanks!